src/backend/src/prisma/migrations/20240207185652_renamed_lead_manager_wbselement/migration.sql

```diff
@@ -0,0 +1,24 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `projectLeadId` on the `WBS_Element` table. All the data in the column will be lost.
+  - You are about to drop the column `projectManagerId` on the `WBS_Element` table. All the data in the column will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "WBS_Element" DROP CONSTRAINT "WBS_Element_projectLeadId_fkey";
```

Comment:
we don't wanna do this this way cause that's gonna lose all the old project manager/ lead data that we had in the database. This will require doing a custom migration, so I'm gonna follow up with u in slack ab this